src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -222,28 +220,6 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
     return null;
   }
 
-  // NOTE(vikram): This is a compliated bit of nonsense code. Because the
```

Comment:
@agiron123, do you have thoughts on the tradeoffs of shrinking the viewport when we open the drawer vs. not? I'm happy to be rid of this code, but I did think that was a slick feature.

That said, I didn't really notice it in the demo, so I don't have a super strong opinion.

Reply:
The code here was adding a bottom offset to the page content, so every time the drawer was opened from the side, content was shifted upwards, which didn't make much sense anymore.

What we can do instead is change this to shift page content to the left every time that the drawer opens from the right side.